templates/taxi/car_detail.html

```diff
@@ -19,4 +19,13 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if user not in car.drivers.all %}
+  <a href="{% url 'taxi:car-driver-assign' pk=car.id %}" class="btn btn-primary">
+    Assign me to this car
+  </a>
+  {% else %}
+  <a href="{% url 'taxi:car-driver-assign' pk=car.id %}" class="btn btn-danger">
+    Delete me from this car
+  </a>
+  {% endif %}
```

Comment:
The logic for assigning or removing the user from the car's drivers list is correctly implemented using an if-else block. However, ensure that the URL name 'taxi:car-driver-assign' matches the intended functionality, as it is used for both assigning and removing the user. Consider using more descriptive button text to clearly indicate the action being performed.